en_US/durability/durability_introduction.md

```diff
@@ -0,0 +1,132 @@
+# Durable MQTT Sessions and Messages
+
+Starting from release v5.7, EMQX contains an embedded durable storage for MQTT sessions and messages.
+This page gives a high-level introduction of the session durability feature in EMQX and how it ensures the resumption of sessions after restart of EMQX nodes.
+
+:::warning
+EMQX v5.7.0 don't support shared subscriptions for the durable sessions yet.
+This feature will be implemented in a later release.
+:::
+
+## Types of Client Sessions in EMQX
+
+According to the MQTT standard, client sessions facilitate the management of client connections and states within the MQTT broker.
+Informally, EMQX separates client sessions into two logical categories:
+
+- **Persistent**: persistent sessions are kept by the broker after the client's connection terminates, and can be resumed if the client reconnects to the broker within the session expiry interval. Messages sent to the topics while the client was offline are delivered.
+- **Ephemeral**: ephemeral sessions exist only for the duration of the client's connection to EMQX. When a client with an ephemeral session disconnects, all session information, including subscriptions and undelivered messages, is discarded.
+
+The client session is considered persistent in two cases:
+
+1. For the clients using MQTT 5 protocol,
+   [Session Expiry Interval](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901048) property of `CONNECT` or `DISCONNECT` packet is set to a value greater than zero.
+
+2. For the clients using MQTT 3.* protocol,
+   [Clean Session](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718030) flag is set to 0,
+   and `mqtt.session_expiry_interval` configuration parameter is set to a value greater than 0.
```

Comment:
Seems mqtt.session_expiry_interval is a parameter in mqtt v5.

Reply:
For V5 client this config doesn't override `Session-Expiry-Interval` property supplied by the client: https://github.com/emqx/emqx/blob/02097b577cdc4efca609deb06c0498eda05482d1/apps/emqx/src/emqx_channel.erl#L1537